proto/ecosystem-incentive/ecosystem_incentive.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+package ununifi.ecosystemincentive;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/UnUniFi/chain/x/ecosystem-incentive/types";
+
+message IncentiveUnit {
+  string id = 1 [
+    (gogoproto.moretags) = "yaml:\"id\""
+  ];
+  repeated SubjectInfo subject_info_list = 2 [
+    (gogoproto.moretags) = "yaml:\"subject_info_lists\"",
+    (gogoproto.nullable) = false
+  ];
+}
+
+message SubjectInfo {
+  string address = 1 [
+    (gogoproto.moretags) = "yaml:\"subject_addr\"",
```

Comment:
more tag and name mismatch - subject_addr and address

Reply:
modified it.

Reply:
modified it.